File: two-sum/two-sum.cs
public class Kata
{
  public static int[] TwoSum(int[] numbers, int target) {
    int[] output =  new int[2];
    for (int i = 0; i < numbers.Length - 1; i++) {
        for (int j = i + 1; j < numbers.Length; j ++) {
            if (numbers[i] + numbers[j] == target) {
                output[0] = i;
                output[1] = j;
            }
         
        }
    }
    return output;
  }
}

File: two-sum.py
def two_sum(numbers, target):
    for x in (numbers[:-1]):
        print(x)
        for y in (numbers[numbers.index(x):]):
            print(y)
            if x + y == target:
                return [numbers.index(x), numbers.index(y, numbers.index(x) + 1)]


print(two_sum([2, 2, 3], 4));